.version

```diff
@@ -1 +1 @@
-v1.7.0-7651345*v1.7*1*7*0*0*7651345
\ No newline at end of file
+v1.8.0-7651345*v1.8*1*8*0*0*7651345
```

Comment:
Commit hash should be `bb7ff67`? help me double check this

Reply:
https://github.com/awslabs/aws-iot-device-client/commit/7651345c55c53ff13fef33e0c35904c6e8b721a4

that was the commit before you update the version for 1.7 @BolongZhang-AWS

Reply:
yeah but for this one we should use the latest commit, head of main?